extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -115,7 +115,8 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
 
     // URL will be validated and resolved based on the canonical URL if relative
     // when navigating.
-    const href = this.element.getAttribute('href');
+    const href =
```

Comment:
Nit: This variable could be referring to an attribute value (str) or an Element, can you update the name so it's clear that it's a boolean (eg: `isRemote` / `hasHref`)

Reply:
Reverted, needed to rebase.